docs/en/base/forms.md

```diff
@@ -34,15 +38,19 @@ View example of an input element
 
 Note: The attribute `readonly` disables the input but it still retains a default cursor.
 
-### Checkbox and radio button
+### Checkbox
 
-Use checkboxes and radio buttons to select one or more options.
+Use checkboxes to select one or more options, default checkboxes can appear in two states: selected and disabled, or unselected.
```

Comment:
Aren't there 4 states? Both selected, unselected have enabled and disabled elements

Reply:
This is valid. There are more then two states as Karl says.